pyrasterframes/src/main/python/docs/getting-started.pymd

```diff
@@ -6,7 +6,7 @@ If you are new to Earth-observing imagery, you might consider looking at the [Co
 
 @@@
 
-There are @ref:[several ways](getting-started.md#other-options) to use RasterFrames, and @ref:[several languages](languages.md) with which you can use it.  
+There are @ref:[several ways](getting-started.md#other-options) to use RasterFrames, and @ref:[several languages (Scala, SQL)](languages.md) with which you can use it.  
```

Comment:
Perhaps here we say `(Python, SQL, or Scala)` as we are yet pretty early on in docs.

Or may be say `... and you can use it in Python, SQL, or Scala`.